lib/plugins/aws/provider/awsProvider.js

```diff
@@ -420,14 +509,79 @@ class AwsProvider {
               },
             },
             memorySize: { $ref: '#/definitions/awsLambdaMemorySize' },
-            resourcePolicy: {
+            notificationArns: { type: 'array', items: { $ref: '#/definitions/awsArnString' } },
+            profile: { type: 'string' },
+            region: {
+              enum: [
+                'us-east-1',
+                'us-east-2',
+                'us-gov-east-1',
+                'us-gov-west-1',
+                'us-west-1',
+                'us-west-2',
+                'af-south-1',
+                'ap-east-1',
+                'ap-northeast-1',
+                'ap-northeast-2',
+                'ap-northeast-3',
+                'ap-south-1',
+                'ap-southeast-1',
+                'ap-southeast-2',
+                'ca-central-1',
+                'eu-central-1',
+                'eu-north-1',
+                'eu-south-1',
+                'eu-west-1',
+                'eu-west-2',
+                'eu-west-3',
+                'me-south-1',
+                'sa-east-1',
```

Comment:
It seems to miss some regions, e.g. those in China or Canada.

Reference https://docs.aws.amazon.com/general/latest/gr/rande.html